### DISCUSSION TITLE
Any way to locate which script causing trouble?

### DISCUSSION THREAD

**Discussion by 117649** [Aris-t2/CustomJSforFx#70] [Q&A]
I'm getting very terrible performance issue on the UI.
Is there any tools to identify which script or extension causing trouble?

**Comment by BoomerangAide** [comment]
Commenting out scripts in [userChrome.js](https://github.com/Aris-t2/CustomJSforFx/blob/master/method%202/profile/userChrome.js) and see what difference it makes?

**Comment by Aris-t2** [comment]
Anything new here?
